.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--- Title your PR with a Jira reference (if available) followed by brief description - for example: "PAYARA-1234 Add readme file" -->
+
+## Description
+This is a bug fix / feature. <!-- delete/modify as applicable-->
+
+<!-- Provide some context here -->
+
+<!--- Please provide enough information here about the what and why of your change. Target for developers of any experience level to understand -->
+
+## Important Info
+
+### Dependant PRs (if applicable)
+<!--- Link any related or dependant PRs here with brief description why -->
+
+### Blockers (if applicable)
+<!-- Detail any blockers with links/info -->
+
+### Deadline (if applicable)
```

Comment:
Does anything have a deadline which isn't simply covered by the milestone field?

Reply:
There's a low chance of it being needed, but it provides thought. It could be a fix for a date related bug, or it could be blocking another issue which needs time to be tested. There's a few niche cases.

Reply:
I'm still not convinced personally. 
We can track all that in Jira and in internal chat -  we don't need a, frankly, dead field in a PR description which isn't going to work as any sort of warning unless you literally read the PR description everyday.